src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -126,19 +132,15 @@ const MultiRangeSlider = ({
     // We always obtain a value lower than the maximum selected, since the maximum is reserved for the input maxVal
     const minValue = Math.min(val, maxVal - 1)
     setMinVal(minValue)
-    if (minValRef.current) {
-      minValRef.current.value = minValue.toString()
-    }
+    if (minValRef.current) minValRef.current.value = minValue.toString()
     maxValRef.current && updateBar(maxValRef.current, range, minValue, min, max)
   }
 
   const updateMax = (val: number) => {
     // We always obtain a value greater than the selected minimum, since the minimum is reserved for the input minVal
     const maxValue = Math.max(val, minVal + 1)
```

Comment:
I think that if it was not used in its multi variant, it should allow to reach 0

Reply:
Nice catch! I agree